2023-03-14 14:57:00,728 INFO    MainThread:30392 [wandb_setup.py:_flush():76] Loading settings from C:\Users\jaydenpark\.config\wandb\settings
2023-03-14 14:57:00,728 INFO    MainThread:30392 [wandb_setup.py:_flush():76] Loading settings from C:\Users\jaydenpark\Documents\GitHub\disaster_tweet\wandb\settings
2023-03-14 14:57:00,728 INFO    MainThread:30392 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-03-14 14:57:00,728 INFO    MainThread:30392 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-03-14 14:57:00,728 INFO    MainThread:30392 [wandb_init.py:_log_setup():428] Logging user logs to C:\Users\jaydenpark\Documents\GitHub\disaster_tweet\wandb\run-20230314_145700-1a14n08j\logs\debug.log
2023-03-14 14:57:00,728 INFO    MainThread:30392 [wandb_init.py:_log_setup():429] Logging internal logs to C:\Users\jaydenpark\Documents\GitHub\disaster_tweet\wandb\run-20230314_145700-1a14n08j\logs\debug-internal.log
2023-03-14 14:57:00,728 INFO    MainThread:30392 [wandb_init.py:_jupyter_setup():378] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001EDE91F4BB0>
2023-03-14 14:57:00,728 INFO    MainThread:30392 [wandb_init.py:init():462] calling init triggers
2023-03-14 14:57:00,728 INFO    MainThread:30392 [wandb_init.py:init():465] wandb.init called with sweep_config: {}
config: {'learning_rate': 1e-07, 'data_size': 1, 'batch_size': 128}
2023-03-14 14:57:00,728 INFO    MainThread:30392 [wandb_init.py:init():515] starting backend
2023-03-14 14:57:00,728 INFO    MainThread:30392 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=spawn, using: spawn
2023-03-14 14:57:00,743 INFO    MainThread:30392 [backend.py:ensure_launched():217] starting backend process...
2023-03-14 14:57:00,817 INFO    MainThread:30392 [backend.py:ensure_launched():222] started backend process with pid: 30432
2023-03-14 14:57:00,817 INFO    MainThread:30392 [wandb_init.py:init():525] backend started and connected
2023-03-14 14:57:00,817 INFO    MainThread:30392 [wandb_run.py:_label_probe_notebook():1089] probe notebook
2023-03-14 14:57:00,817 INFO    MainThread:30392 [wandb_run.py:_label_probe_notebook():1099] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-03-14 14:57:00,817 INFO    MainThread:30392 [wandb_init.py:init():596] updated telemetry
2023-03-14 14:57:00,817 INFO    MainThread:30392 [wandb_init.py:init():627] communicating run to backend with 30 second timeout
2023-03-14 14:57:02,344 INFO    MainThread:30392 [wandb_run.py:_on_init():1924] communicating current version
2023-03-14 14:57:02,393 INFO    MainThread:30392 [wandb_run.py:_on_init():1928] got version response upgrade_message: "wandb version 0.13.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-03-14 14:57:02,393 INFO    MainThread:30392 [wandb_init.py:init():662] starting run threads in backend
2023-03-14 14:57:02,657 INFO    MainThread:30392 [wandb_run.py:_console_start():1898] atexit reg
2023-03-14 14:57:02,658 INFO    MainThread:30392 [wandb_run.py:_redirect():1771] redirect: SettingsConsole.WRAP
2023-03-14 14:57:02,658 INFO    MainThread:30392 [wandb_run.py:_redirect():1808] Wrapping output streams.
2023-03-14 14:57:02,659 INFO    MainThread:30392 [wandb_run.py:_redirect():1832] Redirects installed.
2023-03-14 14:57:02,659 INFO    MainThread:30392 [wandb_init.py:init():688] run started, returning control to user process
2023-03-14 14:57:02,660 INFO    MainThread:30392 [wandb_init.py:_pause_backend():341] pausing backend
2023-03-14 14:57:02,660 INFO    MainThread:30392 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2023-03-14 14:57:02,665 INFO    MainThread:30392 [wandb_init.py:_resume_backend():350] resuming backend
2023-03-14 14:57:02,668 INFO    MainThread:30392 [wandb_init.py:_pause_backend():341] pausing backend
2023-03-14 14:57:02,668 INFO    MainThread:30392 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2023-03-14 14:57:02,683 INFO    MainThread:30392 [wandb_init.py:_resume_backend():350] resuming backend
2023-03-14 14:57:02,872 INFO    MainThread:30392 [wandb_init.py:_pause_backend():341] pausing backend
2023-03-14 14:57:02,872 INFO    MainThread:30392 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2023-03-14 14:57:02,875 INFO    MainThread:30392 [wandb_init.py:_resume_backend():350] resuming backend
2023-03-14 14:57:10,121 INFO    MainThread:30392 [wandb_init.py:_pause_backend():341] pausing backend
2023-03-14 14:57:10,121 INFO    MainThread:30392 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2023-03-14 14:57:10,136 INFO    MainThread:30392 [wandb_init.py:_resume_backend():350] resuming backend
2023-03-14 14:57:12,222 INFO    MainThread:30392 [wandb_init.py:_pause_backend():341] pausing backend
2023-03-14 14:57:12,222 INFO    MainThread:30392 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2023-03-14 14:57:12,225 INFO    MainThread:30392 [wandb_init.py:_resume_backend():350] resuming backend
2023-03-14 14:57:17,696 INFO    MainThread:30392 [wandb_run.py:_config_callback():1136] config_cb None None {'vocab_size': 30522, 'learning_rate': 0.0003}
2023-03-14 15:29:42,697 INFO    MainThread:30392 [wandb_init.py:_pause_backend():341] pausing backend
2023-03-14 15:29:42,697 INFO    MainThread:30392 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2023-03-14 15:29:52,823 INFO    MainThread:30392 [wandb_init.py:_resume_backend():350] resuming backend
2023-03-14 15:29:52,825 INFO    MainThread:30392 [wandb_init.py:_pause_backend():341] pausing backend
2023-03-14 15:29:52,826 INFO    MainThread:30392 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2023-03-14 15:30:46,001 INFO    MainThread:30392 [wandb_run.py:_atexit_cleanup():1867] got exitcode: 0
2023-03-14 15:30:46,002 INFO    MainThread:30392 [wandb_run.py:_restore():1839] restore
2023-03-14 15:30:46,555 INFO    MainThread:30392 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 592
  total_bytes: 592
}

2023-03-14 15:30:46,850 INFO    MainThread:30392 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 592
  total_bytes: 592
}

2023-03-14 15:30:47,816 INFO    MainThread:30392 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 592
  total_bytes: 14456
}

2023-03-14 15:30:47,926 INFO    MainThread:30392 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 592
  total_bytes: 14456
}

2023-03-14 15:30:48,033 INFO    MainThread:30392 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 592
  total_bytes: 14456
}

2023-03-14 15:30:48,144 INFO    MainThread:30392 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 592
  total_bytes: 14456
}

2023-03-14 15:30:48,259 INFO    MainThread:30392 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5998
  total_bytes: 14456
}

2023-03-14 15:30:48,360 INFO    MainThread:30392 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 14456
  total_bytes: 14456
}

2023-03-14 15:30:48,462 INFO    MainThread:30392 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 14456
  total_bytes: 14456
}

